js/paramviewer/ParamViewer.svelte

```diff
@@ -66,6 +66,21 @@
 			}
 		});
 	}
+
+	function render_links(description: string): string {
```

Comment:
This is not going to provide robust protection against XSS, but I think that's okay given the context here.